src/app/(core)/groups/(overview)/page.tsx

```diff
@@ -1,3 +1,171 @@
+'use client';
+import React, { useState, useEffect } from 'react';
+import GroupCard from '@/components/GroupCard';
+import { createClient } from '../../../../utils/supabase/client';
+import Image from 'next/image';
+import Button from '@/components/ui/button';
+import { HiMagnifyingGlass } from "react-icons/hi2";
+import BellIcon from "../../../../../public/assets/icons/BellIcon.svg";
+import MyGroups from '@/components/MyGroups';
+import { FaChevronRight, FaChevronLeft } from 'react-icons/fa';
+import { PiUsersFour } from "react-icons/pi";
+import { CiCalculator1 } from "react-icons/ci";
+import { HiOutlineBookOpen } from "react-icons/hi";
+import { PiGlobeStand } from "react-icons/pi";
+import { PiLaptop } from "react-icons/pi";
+import { LuLanguages } from "react-icons/lu";
+import Science from "../../../../../public/assets/icons/Science.svg";
+import { getLoggedInUser } from '@/utils/actions/user.actions';
+
+const Subjects = [
```

Comment:
idk why you give a variable name in Camel case,

Reply:
What "idk" means?
Answering your question:

Here is a step-by-step guide that will help you remember all of the essential steps of writing a code review.

**How to communicate?**

- Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it?
- Remember to explain why you request a change - it might be obvious for you but not for someone else.
- Remember to **be polite** and clear in your comments - your job here is to help another person improve in a friendly way. ALWAYS use the magic word "Please..."
- You can include emojis and gifs to make your code review more human-friendly 👽.

Intead of <CamelCase> I think I use <PascalCase>, however it is a bad practice for naming an array of objects, here are some tips for naming variables:

In TypeScript, there are no strict rules enforcing how you should name variables, but there are conventions and best practices widely followed in the community to keep the code clean, readable, and consistent. Here are some of the main recommendations:

1. CamelCase for variable and function names:
Use camelCase for naming variables and functions. This means the first word is lowercase, and each subsequent word starts with an uppercase letter.
Example: userName, totalPrice, calculateTotal.
2. PascalCase for classes and types:
Use PascalCase for naming classes, interfaces, types, and enums. Here, each word starts with an uppercase letter.
Example: User, OrderDetails, AddressType.
3. Descriptive and concise:
Choose variable names that are descriptive and clearly represent the purpose of the variable, but without being excessively long.
Example: productCount is better than pCount because it’s clearer.
4. Avoid ambiguous abbreviations:
Avoid abbreviations unless they are widely understood within the context of your project or domain.
Example: It’s preferable to use isAuthenticated instead of isAuth, unless Auth is a commonly accepted term in your code.
5. Constant names in uppercase:
Use UPPER_CASE_SNAKE_CASE for global constants.
Example: MAX_USERS, DEFAULT_TIMEOUT.
6. Prefixes or suffixes for booleans:
It’s a common practice for boolean variables to have a prefix like is, has, should, to clearly indicate that they contain a true or false value.
Example: isLoggedIn, hasAccess, shouldRetry.
7. Avoid single-character variable names:
Unless it’s in the context of a short loop or a mathematical function, it’s better to avoid names like x, y, z, a, etc. Use names that explain the purpose of the variable.
Example: Use index instead of i, unless in a simple loop.
8. TypeScript-specific conventions:
Interfaces: Although it’s an optional convention, sometimes the letter I is used as a prefix for naming interfaces (e.g., IUser). However, it’s not mandatory, and many prefer not to do so.
Optional types: When a property is optional, it’s customary to mark it with a ? (e.g., age?: number), which makes it clear that the property may not be present.
These are some standard practices that will help you write clearer code and maintain consistency in your TypeScript projects.

In this case this variable is outside of the function so it is a constant variable and I should use <UPPERCASE>.